mayastor-test/e2e/install/install_test.go

```diff
@@ -90,6 +101,26 @@ func installMayastor() {
 		"120s", // timeout
 		"1s",   // polling interval
 	).Should(Equal(3))
+
+	Eventually(moacReadyPodCount(),
+		"60s", // timeout
+		"1s",  // polling interval
+	).Should(Equal(1))
+
+	// Now create pools on all nodes.
+	// Note the disk for use on each node has been set in deploy/pool.yaml
+	nodeList := corev1.NodeList{}
+	if (k8sClient.List(context.TODO(), &nodeList, &client.ListOptions{}) != nil) {
+		fmt.Println("Failed to list Nodes, pools not created")
+		return
+	}
+	for _, k8node := range nodeList.Items {
+		bashcmd := "NODE_NAME=" + k8node.Name + " envsubst < " + "pool.yaml" + " | kubectl apply -f -"
```

Comment:
You'll probably want to move the envsubts/kubectl combination out into a library soon.

Reply:
Noted.